src/Result.js

```diff
@@ -72,8 +67,67 @@ class Result extends Component {
         address: result.location.display_address.join(' '),
         tags: sCategories,
         price: result.price,
-        hours: 'HOURS',
+        hours: sHours,
+        description: result.description,
       });
+      const map = document.getElementById('map-image');
```

Comment:
So excited that this turned out not to be deprecated! I think we've got a little CSS work to do for an XS screen still (the elements are a little curshed) but I will have time to mess around with that tomorrow.